File: TransCoder_tokenized_test_set_functions/python/COUNT_ZEROS_IN_A_ROW_WISE_AND_COLUMN_WISE_SORTED_MATRIX.py
def countZeroes ( mat ) :
    N = 5;
    
    row = N - 1;
    
    col = 0;
    
    count = 0;
    
    while ( col < N ) :
        while ( mat [ row ] [ col ] ) :
            if ( row < 0 ) :
                return count;
                
            row = row - 1;
            
        count = count + ( row + 1 );
        
        col = col + 1;
        
    return count;

File: results/java_codex/COUNT_ZEROS_IN_A_ROW_WISE_AND_COLUMN_WISE_SORTED_MATRIX.java
int countZeroes(int mat[][]){
        int row = N-1, col = 0;
        int count = 0;
        while(col < N){
            while(mat[row][col] > 0) if(--row < 0) return count;
            count += (row+1);
            col++;
        }
        return count;
    }